caddy/generate-caddy-file

```diff
@@ -30,14 +30,42 @@ jwt {
 """
 
 
+estimate_comparison_diseases = ["HepB",
+                                "Hib",
+                                "HPV",
+                                "JE",
+                                "Measles",
+                                "MenA",
+                                "PCV",
+                                "Rota",
+                                "Rubella",
+                                "YF"]
+
+estimate_comparison_template = """
+jwt {
+    path /estimate-comparison/201810/${name}
+    allow ${name} reviewer
```

Comment:
Given we don't need to distinguish members from reviewers in the end shall we stick with the simpler `allow ${diseaseName} true`, analogous to the  `allow ${modellingGroupName} true` rules for model-documentation?

Reply:
I was just following https://vimc.slack.com/archives/G72HCRX5L/p1550151212032700 but I see the simpler one you wrote here: https://vimc.slack.com/archives/G72HCRX5L/p1550151271034500 - let's do that instead

Reply:
Yeah I was mistaken about the required level of complexity at first.